data/Makefile

```diff
@@ -948,12 +987,27 @@ g=$(shell awk '{x += $$2} END{print x}' $(name)/$(ref).fa.fai)
 %.n$n.tsv: %.tsv
 	mlr --tsvlite filter '$$n >= $n' $< >$@
 
+# Filter 10% of lowest weighted edges using python.
+%.m10.tsv: %.tsv
+	$(time) $(python) $(bin)/physlr filter-overlap --minimizer-overlap 10 $< >$@
+
+# Filter 20% of lowest weighted edges using python.
+%.m20.tsv: %.tsv
+	$(time) $(python) $(bin)/physlr filter-overlap --minimizer-overlap 20 $< >$@
+
+# Filter 50% of lowest weighted edges using python.
+%.m50.tsv: %.tsv
+	$(time) $(python) $(bin)/physlr filter-overlap --minimizer-overlap 50 $< >$@
+
```

Comment:
Just wondering why we need these explicit rules when we have the rule using the variable `m`?

Reply:
these explicit rules are legacies of the past. I can remove them though.

Reply:
I'm OK with removing them - doesn't seem necessary to me